### ISSUE TITLE
Update bell charts to work on plan pages

### ISSUE THREAD

**Issue by migurski** [PlanScore/PlanScore#371]
Follow-on to https://github.com/PlanScore/FrontPage/pull/11, the bell charts appear incorrect on plan pages e.g.

<img width="1279" alt="Screen Shot 2021-06-29 at 5 09 10 PM" src="https://user-images.githubusercontent.com/58730/123882936-bb42af80-d8fc-11eb-899a-8dec27baff32.png">

https://dev.planscore.org/plan.html?20210602T011319.378415404Z

**Comment by migurski** [comment]
Fixed this in https://github.com/PlanScore/FrontPage/commit/fdb550db11eeb2521e2226c2a6a7ff18558db7a5

**Comment by paulirish** [comment]
> Fixed this in [PlanScore/FrontPage@fdb550d](https://github.com/PlanScore/FrontPage/commit/fdb550db11eeb2521e2226c2a6a7ff18558db7a5)

Thanks for fixing this!  Apologies for not getting to it earlier. 

When I made the PR, I thought I had tested it, but apparently I was not realizing the subtle distinction between the various index.css's in Frontpage/ and its sitewide.css build. ;)  

Ultimately moving all those bits into sitewide seems great. Makes this stylesharing situation much easier to reason about, so big 👍 from me.


Going forward I'll be using this trick when working on #368: 


![image](https://user-images.githubusercontent.com/39191/124329019-8ecaa580-db3f-11eb-8e1d-834d4607c91d.png)


Cheers
